src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -209,6 +224,18 @@ public void confirm(CorrelationData correlationData, boolean ack, String cause)
         return rabbitTemplate;
     }
 
+    public String getQueueName() {
+        String durableName = queueDurable ? "durable" : "transient";
+        return domainId + "." + componentName + "." + consumerName + "." + durableName;
+    }
+
+    public String getWaitlistQueueName() {
+
+        String durableName = queueDurable ? "durable" : "transient";
+        return domainId + "." + componentName + "." + consumerName + "." + durableName + "."
+                + waitlistSufix;
```

Comment:
Is it some extra new line endings here?

Reply:
I'm using line length 100 so if I format a document it will linebreak.